course02/project01/gavin/Asteroids/application.cc

```diff
@@ -0,0 +1,33 @@
+//
+
+#include "application.h"
+
+//
+// Created by gavinparker on 27/12/18.
+Application::Application() : mGameWorld(mController),
+                            mLastFrameTime(static_cast<float>(getElapsedSeconds())),
+                            mThisFrameTime(static_cast<float>(getElapsedSeconds())){
+
+}
+
+void Application::draw()
+{
+    mThisFrameTime = static_cast<float>(getElapsedSeconds());
```

Comment:
This should be a strong type e.g. `std::chrono::seconds`, not a `float`. The time unit is thus explicit and carried with the type.

Reply:
Fixed in 76db252356e796ace0396de1cb5a68c9790e3d06, with a floating-point duration rep.